src/Features/CSharp/Portable/MakeFieldReadonly/CSharpMakeFieldReadonlyDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.MakeFieldReadonly;
+
+namespace Microsoft.CodeAnalysis.CSharp.MakeFieldReadonly
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal class CSharpMakeFieldReadonlyDiagnosticAnalyzer :
+        AbstractMakeFieldReadonlyDiagnosticAnalyzer<IdentifierNameSyntax, ConstructorDeclarationSyntax>
+    {
+        protected override void InitializeWorker(AnalysisContext context)
+            => context.RegisterSyntaxNodeAction(AnalyzeType, SyntaxKind.ClassDeclaration, SyntaxKind.StructDeclaration);
+
+        protected override ISyntaxFactsService GetSyntaxFactsService() =>
```

Comment:
please place => on the next line (indented).

Reply:
➡️ Added in commit 38db528